spec/support/contexts/response_tree.rb

```diff
@@ -190,12 +169,4 @@ def expect_read_only_value(path, value)
     el = page.find("[data-path='#{path.join('-')}']")
     expect(el).to have_content(value)
   end
-
-  def temporal_mapping
-    {year: "1i", month: "2i", day: "3i", hour: "4i", minute: "5i", second: "6i"}
-  end
-
-  def control_for_temporal(path, type, subfield)
-    find("##{path_selector(path, "#{type}_value_#{temporal_mapping[subfield]}")}")
```

Comment:
we don't need any of this anymore because we no longer use subfields for times and dates